content/versions-comparison/4.0-news.json

```diff
@@ -63,14 +56,14 @@
     "link": "https://api.akeneo.com/api-reference-index.html#AssetManager"
   },
   {
-    "title":"Quality grades on simple product 💯",
+    "title":"Data quality grades on products 💯",
     "domains": ["Quality", "Reporting"],
     "editions": ["EE"],
     "since-version": "4.0",
     "link": "/pim/v4/updates/data-quality-insights.html#monitor-your-data-quality-in-a-dashboard"
   },
   {
-    "title":"Data quality monitoring in a dashboard 📊",
+    "title":"Data Quality Insights dashboard 📊",
```

Comment:
Again, the feature's name does give any clue on what it is for people who never heard about it.